theme_yes/views/images.xml

```diff
@@ -284,5 +284,16 @@ Check in theme_kea's primary_variables.scss, theme.scss and theme_common's mixin
     <field name="name">website.s_accordion_image_default_image</field>
     <field name="url">/theme_yes/static/src/img/snippets/s_banner.jpg</field>
 </record>
+<!-- // Intro Pill // -->
+<record id="s_intro_pill_default_image" model="theme.ir.attachment">
+    <field name="key">website.s_intro_pill_default_image</field>
+    <field name="name">website.s_intro_pill_default_image</field>
+    <field name="url">/theme_vehicle/static/src/img/snippets/s_intro_pill_default_image.jpg</field>
+</record>
+<record id="s_intro_pill_default_image_2" model="theme.ir.attachment">
+    <field name="key">website.s_intro_pill_default_image_2</field>
+    <field name="name">website.s_intro_pill_default_image_2</field>
+    <field name="url">/theme_vehicle/static/src/img/snippets/s_intro_pill_default_image_2.jpg</field>
```

Comment:
Errr... strange car pictures for weddings ^_^; It seems this copy/paste was not adapted.
I noticed it by pure chance - I'll see to add a test to verify we have no such accident in other blocks.

Reply:
Well spotted, I double checked the rest and it should be fine now